packages/app-store/paypal/api/capture.ts

```diff
@@ -0,0 +1,69 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import z from "zod";
+
+import { findPaymentCredentials } from "@calcom/app-store/paypal/api/webhook";
+import Paypal from "@calcom/app-store/paypal/lib/Paypal";
+import { IS_PRODUCTION } from "@calcom/lib/constants";
+import { getErrorFromUnknown } from "@calcom/lib/errors";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
This will be used if user is redirected to our page successfully to validate payment right away and redirect accordingly.